### ISSUE TITLE
Missing language localization for "Columns"

### ISSUE THREAD

**Issue by dotek** [wenzhixin/bootstrap-table#6331]
### Bootstraptable version(s) affected

1.20

### Description

The language localization of the "Columns" value for the aria-label attribute is missing.

<code>aria-label=\"Columns\" title=\"").concat(opts.formatColumns(), "\"</code>

See: https://github.com/wenzhixin/bootstrap-table/blob/develop/dist/bootstrap-table.js#L6022

### Example(s)

![bt-ml](https://user-images.githubusercontent.com/274314/184596272-c25218b3-3c7f-4dc1-a8d3-5b20cbe54567.png)
